tests/integration/tables/crashes.cpp

```diff
@@ -22,35 +22,30 @@ class crashes : public testing::Test {
 };
 
 TEST_F(crashes, test_sanity) {
-  // 1. Query data
-  auto const data = execute_query("select * from crashes");
-  // 2. Check size before validation
-  // ASSERT_GE(data.size(), 0ul);
-  // ASSERT_EQ(data.size(), 1ul);
-  // ASSERT_EQ(data.size(), 0ul);
-  // 3. Build validation map
-  // See helper.h for avaialbe flags
-  // Or use custom DataCheck object
-  // ValidationMap row_map = {
-  //      {"type", NormalType}
-  //      {"pid", IntType}
-  //      {"path", NormalType}
-  //      {"crash_path", NormalType}
-  //      {"identifier", NormalType}
-  //      {"version", NormalType}
-  //      {"parent", IntType}
-  //      {"responsible", NormalType}
-  //      {"uid", IntType}
-  //      {"datetime", NormalType}
-  //      {"crashed_thread", IntType}
-  //      {"stack_trace", NormalType}
-  //      {"exception_type", NormalType}
-  //      {"exception_codes", NormalType}
-  //      {"exception_notes", NormalType}
-  //      {"registers", NormalType}
-  //}
-  // 4. Perform validation
-  // validate_rows(data, row_map);
+  ValidationMap row_map = {
+      {"type", NormalType},
+      {"pid", IntType},
+      {"path", NormalType},
+      {"crash_path", NormalType},
+      {"identifier", NormalType},
+      {"version", NormalType},
+      {"parent", IntType},
+      {"responsible", NormalType},
+      {"uid", IntType},
+      {"datetime", NormalType},
+      {"crashed_thread", IntType},
+      {"stack_trace", NormalType},
+      {"exception_type", NormalType},
+      {"exception_codes", NormalType},
+      {"exception_notes", NormalType},
+      {"registers", NormalType},
+  };
+
+  auto const data = execute_query(
+      "select crashes.* from users CROSS JOIN crashes "
```

Comment:
This seems flaky, we cannot assure there is a crash on a machine.

Reply:
Could be. I think it's worth the joined query, but I agree if it proves flakey we should not require non-empty. As with others, I PR'ed something to see what it'd do